viewer/src/views/threejs/cad/RootSectorNode.ts

```diff
@@ -69,6 +76,8 @@ export class RootSectorNode extends SectorNode {
           throw new Error(`Unsupported level of detail ${sector.levelOfDetail}`);
       }
     })();
+    group.userData.cached = true;
+    group.userData.used = true;
```

Comment:
Could we add some functions to `SectorNode`, e.g. `increaseRefCount()`, `isReferenced()` and `decreaseRefCount()`?

Reply:
SectorNode is not known in the cache, but I can change to using a refCount in userData?